centreon/src/Core/Domain/Common/GeoCoords.php

```diff
@@ -49,6 +52,11 @@ public function __construct(
         public readonly string $latitude,
         public readonly string $longitude,
     ) {
+        Assertion::maxLength($latitude . ',' . $longitude, self::MAX_LENGTH, 'GeoCoords::maxLength');
```

Comment:
I would have check the total length after the format of the string (but this is more personnal preference)
Also the auto truncate of the number of decimal should happen after the format validation to prevent strange errors in case of the format not being respected